### PR TITLE
#68: Submit buttons should be in 'actions' wrapper 

### LINKED ISSUE #68: Submit buttons should be in 'actions' wrapper
To take advantage of core button styling, can we wrap submit buttons in an actions container? I can make a PR.

### PR DESCRIPTION
**Pull request by laryn**
Fixes #68

### CODE REVIEW

**Review by klonos — APPROVED** [review]
LGTM 👍🏼

**Comment by klonos** [comment]
@laryn can you please rebase, and allow GHA-based tests to run for this? Thanks

**Comment by laryn** [comment]
@klonos I closed and reopened to get GHA checks to run.

File: i18n_menu/i18n_menu.admin.inc [hunk 1]
```diff
@@ -54,10 +54,16 @@ function i18n_menu_translation_form($form, $form_state, $translation_set = NULL,
       );
     }
   }
-  $form['actions'] = array('#type' => 'actions');
-  $form['actions']['update'] = array('#type' => 'submit', '#value' => t('Save'));
+  $form['actions']['#type'] = 'actions';
```
Comment (indigoxela) [inline i18n_menu/i18n_menu.admin.inc:57]: Confused... :wink: 

```
$form['actions'] = array('#type' => 'actions');
```
That means:  `$form['actions']['#type'] = 'actions';`

What exactly needs to get "fixed" here?

File: i18n_node/i18n_node.pages.inc [hunk 2]
```diff
@@ -160,7 +160,7 @@ function i18n_node_select_translation($form, &$form_state, $node, $translations)
       );
     }
   }
-  $form['actions'] = array('#type' => 'actions');
+  $form['actions']['#type'] = 'actions';
```
Comment (indigoxela) [inline i18n_node/i18n_node.pages.inc:163]: Same as above - here it *is* of type "actions".

File: i18n_menu/i18n_menu.admin.inc (hunk 1 above)
Comment (laryn) [reply i18n_menu/i18n_menu.admin.inc:57]: @indigoxela 
I think I was mainly going for code consistency in formatting rather than a functional change here.

**Comment by indigoxela** [comment]
Trying to trigger GHA workflow by close/open.

**Pull request merged**